pandas/tests/io/formats/test_to_html.py

```diff
@@ -235,6 +235,22 @@ def test_to_html_truncate(datapath):
     assert result == expected
 
 
+def test_to_html_truncate_formatter(datapath):
+    data = [
+        {"A": 1, "B": 2, "C": 3, "D": 4},
+        {"A": 5, "B": 6, "C": 7, "D": 8},
+        {"A": 9, "B": 10, "C": 11, "D": 12},
+        {"A": 13, "B": 14, "C": 15, "D": 16},
+    ]
+
+    df = pd.DataFrame(data)
```

Comment:
```suggestion
    df = DataFrame(data)
```

we don't need the pd here.

or maybe `df = DataFrame(np.arange(16).reshape(4, 4) + 1, columns=list("ABCD"))` which would not require a change to the html file.